app/routes.js

```diff
@@ -156,7 +159,7 @@ export const routes = (authProvider) => {
   });
 
   errorHandler(router, (error, req, res) => {
-    logger.error(`${error.title} - ${error.description}`);
+    logger.error(`${error.title} - ${error.description} - ${JSON.stringify(error)}`);
```

Comment:
if there is no error.title or error.description I think this will cause a problem?

Reply:
I don't remember adding this

Reply:
I think at this point it will have an error.title and error.description. The withCatch will catch all errors and change it to our errorContect model. So this should work

Reply:
if we catch every error then that would be the case but what happens if we miss something?

Reply:
we shouldn't the withCatch will catch anything that we missed.

```
  try {
    return await route(req, res, next);
  } catch (err) {
    const newError = new ErrorContext({
      status: 500,
    });
    return next(newError);
  }
```

Reply:
cool